src/test/java/org/cactoos/package-info.java

```diff
@@ -25,6 +25,14 @@
 /**
  * Tests.
  *
+ * @todo #1396:30min Continue replacing usage of MatcherAssert.assertThat with
+ *  Assertion from cactoos-matchers.
+ *  1) Revert 44540d94
+ *  2) Choose package & inline assertThat in it
+ *  3) Undo 1)
+ *  4) Update puzzle for the next package
```

Comment:
@andreoss Sorry, but isn't very clear why 1) Revert the ` 44540d94` commit and 3) Undo 1. Can you check it is correct and add more information to be clearer?

Reply:
@andreoss I'm not a big fan of that, it seems to be very dependent on the IDE you are using (IDEA I suppose?), not everybody is using that IDE, so as asked by @fabriciofx be clearer on the instructions and say with which IDE it is for.

Reply:
@victornoel @fabriciofx You are right. I reworded it in more general terms. And now it seems that the inlining approach is more convoluted, since it leaves behind many unwanted artifacts (wrong variable names, broken indentation, etc)